pymc4/coroutine_model.py

```diff
@@ -0,0 +1,236 @@
+import biwrap
```

Comment:
Can this module just be called `model.py`, or are we expecting other (non-coroutine) APIs for models?

Reply:
Just tried to be explicit